### PR TITLE
feat: introduce default validators with required one

### PR DESCRIPTION
**Pull request by PierreDemailly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,16 @@ const packageName = await prompt('Package name', {
...
 ```
 
+**This package provide some validators for common usage**
+
+- required
+
+```js
+import { prompt, Validators } from "@topcli/prompts";
+
+const name = await prompt("What's your name ?", { validators: [Validators.required] });
```
Comment (fraxken) [inline README.md:72]: Jump line for the options? (that's quite a lot of code on the same line).

File: src/validators.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export class Validators {
```
Comment (fraxken) [inline src/validators.js:1]: Any technical reason for a class instead of function? 

If you use a class with static, would it make sens to freeze it?

Comment (PierreDemailly) [reply src/validators.js:1]: Zero technical reason.
I've been inspired by Angular validators and wanted to have `Validators.required` syntax ^^
I agree a freeze would be nice, do you think a class with static getters is a wrong choice or it's ok and I just freeze it ?

Comment (fraxken) [reply src/validators.js:1]: It depend, I would use standalone functions (feel like using class is kinda an usage abuse).

Comment (PierreDemailly) [reply src/validators.js:1]: I go for it then 👍

File: README.md (hunk 1 above)
Comment (PierreDemailly) [reply README.md]: I've also updated others snippets. Some snippets were written in `standard` way, it's also updated.

**Review by fraxken — APPROVED** [review]

**Pull request merged**